src/cookbook/editable-svg-icons.md

```diff
@@ -164,4 +164,4 @@ Other tooling to help manage SVG icons includes:
 - [svg-sprite-loader](https://github.com/kisenka/svg-sprite-loader)
 - [svgo-loader](https://github.com/rpominov/svgo-loader)
 
-These tools bundle SVGs at compile time, but make them a little harder to edit during runtime, because `<use>` tags can have strange cross-browser issues when doing anything more complex. They also leave you with two nested `viewBox` properties and thus two coordinate systems. This makes the implementation a little more complex.
+These tools bundle SVGs at compile-time, but make them a little harder to edit during runtime, because `<use>` tags can have strange cross-browser issues when doing anything more complex. They also leave you with two nested `viewBox` properties and thus two coordinate systems. This makes the implementation a little more complex.
```

Comment:
**suggestion**: I disagree with this change: I could easily replace "compile time" with "time of compilation". Yes, Grammarly suggests the hyphen but we shouldn't follow Grammarly suggestions blindly.

Reply:
Thank you so much @NataliaTepluhina for the suggestion, Yeah it makes sense
I've changed "compile time" to "time of compilation"